schemas/authenticate-hawk-response.yml

```diff
@@ -0,0 +1,45 @@
+$schema:  http://json-schema.org/draft-04/schema#
+title:                    "Hawk Signature Authentication Response"
+description: |
+  Response from a request to authenticate a hawk request.
+anyOf:
```

Comment:
I think that would certainly make parsing the response easier -- `"message" in response` doesn't scan as "if the authentication failed"